### PR TITLE
Boatmode

### PR DESCRIPTION
**Pull request by Hwurzburg**
addresses #4072

### CODE REVIEW

**Review by nkruzan — APPROVED** [review]
LGTM

File: common/source/docs/common-advanced-configuration.rst [hunk 1]
```diff
@@ -83,6 +83,7 @@ tuning options for the vehicle.
 [site wiki="copter"]
     Motor Thrust Scaling <motor-thrust-scaling>
 [/site]
+    Moving Vehicle Initialization <common-moving-vehicle-initialization>
```
Comment (rmackay9) [inline common/source/docs/common-advanced-configuration.rst:86]: can we add "(aka Boat Mode)" here and on the page itself?

I wonder if "Moving Vehicle Initialisation" is clear enough.  How about "Starting from a moving platform"?... or "Initialising from a Moving Platform"?

**Review by rmackay9 — APPROVED** [review]
I don't feel strongly enough about my comments to block merging.. :-).

File: common/source/docs/common-advanced-configuration.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-advanced-configuration.rst:86]: if someone searched they would find boat mode and be re-directed....but will change to "Initializing from a Moving Platform"

**Pull request merged**